src/Pos/Wallet/Web/Api.hs

```diff
@@ -250,6 +255,7 @@ instance ToSchema      SyncProgress
 instance ToSchema      ChainDifficulty
 instance ToSchema      BlockVersion
 instance ToSchema      BackupPhrase
+instance ToParamSchema CPassPhrase
 
 -- | Build Swagger-specification from 'walletApi'.
 swaggerSpecForWalletApi :: Swagger
```

Comment:
This `Swagger` stuff is absent in master. Branch cannot be merged because of that.